src/pages/data-science-tools/hadoop/index.md

```diff
@@ -61,6 +61,9 @@ To fix this issue, hadoop keeps redundant data. 3 copies of each block is kept i
 
 To fix the 3rd problem, concept called NameNode standby. There are 2 namenodes used here. One is active, one is standby with the standby taking over when the active namenode undergoes failure.
 
+Hadoop Common
+The third module is Hadoop Common which will provide the tools that are required for the user's computer systems(Windows,Unix) to read data stored under the Hadoop File System.
```

Comment:
Fix spacing "computer systems(Windows,Unix)" to "computer systems (Windows or Unix)" (Did you mean Linux instead of Unix?)